Sources/MotorControl/Model/TwoHandAssessmentObject.swift

```diff
@@ -48,5 +50,133 @@ final class TwoHandAssessmentObject : AbstractAssessmentObject {
     convenience init() {
         self.init(identifier: "example", children: [])
     }
+    
+    override func instantiateNavigator(state: NavigationState) throws -> Navigator {
+        TwoHandNavigator(identifier: identifier, nodes: children)
+    }
+}
+
+final class TwoHandNavigator : Navigator {
+    
+    public let identifier: String
+    public let nodes: [Node]
+    public let handOrder: [HandSelection]
+
+    public init(identifier: String, nodes: [Node]) {
+        guard Set(nodes.map { $0.identifier }).count == nodes.count else {
+            fatalError("identifiers not unique")
+        }
+        self.identifier = identifier
+        self.handOrder = arc4random_uniform(2) == 0 ? [.left, .right] : [.right, .left]
```

Comment:
```suggestion
        let handOrder: [HandSelection] = arc4random_uniform(2) == 0 ? [.left, .right] : [.right, .left]
```

Suggest using a local variable. If you want to track the hand order then you certainly can but use a local variable to do your manipulation and then set it at the end.